types/notifyjs/index.d.ts

```diff
@@ -115,4 +115,4 @@ interface INotifyOption {
      */
     requireInteraction?: boolean;
 }
-export = Notify
+export default Notify
```

Comment:
`export =` is correct but `export default` not.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

```shell
> node -p 'var window = {}; require("notifyjs")'
{ [Function: Notify]
  isSupported: [Function],
  needsPermission: true,
  requestPermission: [Function] }
```

```shell
> node -p 'var window = {}; require("notifyjs").default'
undefined
```